pyproject.toml

```diff
@@ -79,3 +79,50 @@ pylint = ["testutils/testing_pylintrc"]
 
 [tool.setuptools.dynamic]
 version = {attr = "pylint.__pkginfo__.__version__"}
+
+[tool.aliases]
+test = "pytest"
+
+[tool.pytest.ini_options]
+testpaths = ["tests"]
+python_files = ["*test_*.py"]
+addopts = "--strict-markers"
+markers = [
+    "primer_stdlib: Checks for crashes and errors when running pylint on stdlib",
+    "primer_external_batch_one: Checks for crashes and errors when running pylint on external libs (batch one)",
+    "benchmark: Baseline of pylint performance, if this regress something serious happened",
+    "timeout: Marks from pytest-timeout.",
+    "needs_two_cores: Checks that need 2 or more cores to be meaningful",
+]
+
+[tool.isort]
+profile = "black"
+known_third_party = ["platformdirs", "astroid", "sphinx", "isort", "pytest", "mccabe", "six", "toml"]
+skip_glob = ["tests/functional/**", "tests/input/**", "tests/extensions/data/**", "tests/regrtest_data/**", "tests/data/**", "astroid/**", "venv/**"]
```

Comment:
Why are we skipping `astroid/**` here? @Pierre-Sassoulas

Reply:
Wild guess, maybe because of local astroid install that would be affected ?